### DISCUSSION TITLE
Other DML, DDL functions

### DISCUSSION THREAD

**Discussion by MichalisDBA** [osalvador/ReplicaDB#76] [Ideas]
Hello.

I'm just thinking if it would be possible in future release maybe you can implement something to run custom queries against the different databases.

The ReplicaDB acts as a provider, i know it is meant only for sync but as long as you can connect through it in different databases why don't also have the ability to run querries etc. It would be great to do some kind of automations with this.

Thank you.

**Comment by osalvador** [comment]
Hi @MichalisDBA ,

I think there are fantastic open source products that fit perfectly with what you describe.

I usually use `usql` https://github.com/xo/usql to query any database provider and have the same familiar CLI, such as postgres `psql`. 

Is this what you mean?

**Reply by MichalisDBA** [reply]
I think you are right. Didn't know this existed. I will check it for sure. Thanks for the feedback. Your product is amazing and hope you keep working on it.
